internal/integrations/v4/integration/integration.go

```diff
@@ -24,6 +24,7 @@ const (
 	defaultIntegrationInterval = config.FREQ_PLUGIN_EXTERNAL_PLUGINS * time.Second
 	defaultTimeout             = 120 * time.Second
 	minimumTimeout             = 100 * time.Millisecond
+	intervalEnvVarName         = "NRI_CONFIG_INTERVAL"
```

Comment:
Is there a reason why this is not a configuration option? All the config options from config.go are automatically converted also into environment variables. So if we add config_interval, will be converted into NRIA_CONFIG_INTERVAL

Reply:
do you mean `pkg/config.go` ? i think these are Agent configs, this is a env var that is sent to each integration depending on the interval they have configured automagically, no need for the customer to set an extra config on the `newrelic-infra.yml`